hieradata/common.yaml

```diff
@@ -1210,6 +1250,9 @@ router::nginx::robotstxt: |
   User-agent: deepcrawl
   Disallow: /
 
+sidekiq::host: 'redis-1.backend'
```

Comment:
Can someone in @alphagov/team-gov-uk-infrastructure-admins check this is ok please?